### PR TITLE
CLOP-10 cleaner conditional for hosted zones

### PR DESCRIPTION
**Pull request by paulschwarzenberger**

### CODE REVIEW

File: root_locals.tf [hunk 1]
```diff
@@ -24,7 +24,6 @@ locals {
   athena_bucket     = "${var.project}-athena-${local.environment}"
   config_bucket     = "${var.project}-config-${local.environment}"
 
-  hosted_zone_envs    = ["intg", "staging", "prod"]
-  create_domain_email = contains(local.hosted_zone_envs, terraform.workspace) ? true : false
-  create_hosted_zone  = contains(local.hosted_zone_envs, terraform.workspace) ? true : false
+  create_domain_email = var.project == "tdr" ? true : false
```
Comment (TomJKing) [inline root_locals.tf:27]: Is this going to work with the TDR sandbox and management accounts. With the current configuration the domain email and hosted zone are not created in these two environments. With the suggested changes they will be created, which is not desired behaviour.

**Review by TomJKing — COMMENTED** [review]
Just one comment. Might want to double check with Sam as well as to whether this would affect DR2.

File: root_locals.tf (hunk 1 above)
Comment (paulschwarzenberger) [reply root_locals.tf:27]: @TomJKing both the TDR sandbox and management accounts have hosted zones created by this repo - with the previous conditional they would have been destroyed.

Comment (paulschwarzenberger) [reply root_locals.tf:27]: I've logged in to each of the 4 Digital Preservation AWS accounts, and none of them contain a Route53 Hosted Zone

Comment (paulschwarzenberger) [reply root_locals.tf:27]: I've also confirmed the behaviour by doing a Terraform plan of this branch against TDR sandbox and there are no changes planned to hosted zones

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**